tests/drivers/gnss/gnss_api/boards/qemu_x86_64.conf

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Trackunit Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_GNSS_SATELLITES=y
+
+# GPS
+# GPS + GLONASS
+# GPS + GLONASS + QZSS
+# GPS + GLONASS + GALILEO + BEIDOU + QZSS + SBAS
+CONFIG_TEST_ENABLED_SYSTEMS_0=1
+CONFIG_TEST_ENABLED_SYSTEMS_1=3
+CONFIG_TEST_ENABLED_SYSTEMS_2=19
+CONFIG_TEST_ENABLED_SYSTEMS_3=95
+
+# This emmulated board runs real-time
```

Comment:
Expanded comment to read
```
# This emulated board runs real-time
# Lower timeouts to prevent unnecessary waiting
```
and upped the search timeout to 5 seconds from one, to get at least one callback with satellites.

Reply:
I still don't fully grasp, how do the others run then if this one runs real-time?

Reply:
The native sim boards have the config `NATIVE_SIM_SLOWDOWN_TO_REAL_TIME` which one can set to slow them down, so k_msleep(1) takes 1ms. The qemu x86 board runs in simulated time by default, but the 64 bit version runs at real-time. I don't know why this is the case.

Reply:
Okay, thanks for the explanation!